internal/sql/repository/helper/AppListingRepositoryQueryBuilder.go

```diff
@@ -61,7 +61,7 @@ const (
 func (impl AppListingRepositoryQueryBuilder) BuildAppListingQuery(appListingFilter AppListingFilter) string {
 	whereCondition := impl.buildAppListingWhereCondition(appListingFilter)
 	orderByClause := impl.buildAppListingSortBy(appListingFilter)
-	query := "SELECT env.id AS environment_id, env.environment_name, a.id AS app_id, a.app_name,  env.default, p.id as pipeline_id" +
+	query := "SELECT env.id AS environment_id, env.environment_name, a.id AS app_id, a.app_name, env.default, p.id as pipeline_id, env.active" +
```

Comment:
why did we not add the condition env.active = false here? instead we are filtering in code

Reply:
apps which are not linked with any env yet will also filter out in that case, we need them to render in app listing.